service/src/main/java/com/epam/esm/service/entitydtomapper/impl/TagDTOMapperImpl.java

```diff
@@ -0,0 +1,50 @@
+package com.epam.esm.service.entitydtomapper.impl;
+
+import com.epam.esm.dto.CertificateDTO;
+import com.epam.esm.dto.TagDTO;
+import com.epam.esm.entity.Tag;
+import com.epam.esm.service.entitydtomapper.TagDtoMapper;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class TagDTOMapperImpl implements TagDtoMapper {
```

Comment:
i don't see a lot of sense in using both DTOs here, they only bring more complexity

Reply:
reminder: read(response)/write(request) DTOs